GIT_CONFLICTS.md

```diff
@@ -0,0 +1,77 @@
+# Wanderlust Project
+
+## Overview
+Welcome to the Wanderlust project! This is a collaborative effort to build an amazing application. We appreciate your contributions and aim to make the process as smooth as possible.
+
+## Common QnA for Git and Open Source Contributions
+
+<details>
+<summary>
+
+### How can I know if I’m in a bad situation?
+</summary>
+I suggest you to run  **`gitk`**  or  **`gitk --all`**. The commit tree should be self-explainatory.
+
+Also, if  **`git log`**  shows you any merge commits, this means that it isn’t properly synced with the upstream branch
+</details>
+
+### How to Hard Sync Master with Upstream
+To hard sync your local master branch with the upstream master branch, follow these steps in your main branch:
+```
+git remote add upstream /url-to-original-repo
+git fetch upstream
+git reset --hard upstream/main  
+git push origin main --force 
+```
+## Better commits
+
+You should not push what is not needed. Doing  **`git commit -a`**, instead, commits just everything. This, often, is bad.  
+You’d better use  **`git add`**  and, most of all,  **`git add -p`**  

```

Comment:
We are in 3 sec world, people can't hold their eyes more than 3 seconds, to convince them to read, we have to keep it in simple bullet points

Reply:
done